### ISSUE TITLE
Autocomplete search feature with the JetSearch widget 

### ISSUE THREAD

**Issue by Pranavgajjar26** [Crocoblock/suggestions#1356]
Autocomplete feature is not available in crocoblock widget gallery

Autocomplete search feature complete the word or phrase by comparing available search queries with best possible matches. When user has half typed the word, autocomplete gives easiness to finish remaining word or sentence. It should be added to JetSearch widget. 

It should be used along with AJAX search. AJAX show multiple suggestion but it doesn't track or highlight typed characters so user can easily find the best match among the available option. In short it even narrow down the AJAX search results. I have a large database and want to put this widget in the middle of the page and want to bring focus on the search window only so user have to only type part of the sentence and they can easily access to the content without going to the archives.
Please find the available feature link below on one of the plugin provider, I like your services so want to stick to the JetSearch unless I dont have any option.
https://facetwp.com/documentation/facets/facet-types/autocomplete/

**Comment by oceandiveloper** [comment]
yes please. this feature is so important for ecommerce websites for better UX and JetSearch needs to improve in this area.

**Comment by RewindLife** [comment]
I agree and fully support this feature request!!!! Auto-complete in JetSearch would blow away any competition and is kind of expected by users nowadays. Please, please include it into your next update of JetSearch. 
Shchyro diakuyu!

**Comment by aalexandr21** [comment]
You the JetSearch Suggestions https://crocoblock.com/knowledge-base/jetsearch/search-suggestions-widget-overview/
